pyverbs/providers/efa/libefa.pxd

```diff
@@ -17,5 +17,14 @@ cdef extern from 'infiniband/efadv.h':
         unsigned int            device_caps
         unsigned int            max_rdma_size
 
+    cdef struct efadv_ah_attr:
+        unsigned long comp_mask
+        unsigned short ahn
+        unsigned char reserved[6]
```

Comment:
It's preferred if you use  uint64_t, uint16_t etc. as they're defined in C (you can import them from libc.stdint)
But anyway that shouldn't matter as eventually they're linked to the C code by Cython...

Reply:
Ack